packages/utilities/psammead-styles/src/font-families-config.js

```diff
@@ -0,0 +1,12 @@
+const helmet = 'Helmet, Freesans, Helvetica, Arial, sans serif';
+const reithFallback = ', Helvetica, Arial, sans-serif';
+const reithSans = `ReithSans${reithFallback}`;
+const reithSerif = `ReithSerif${reithFallback}`;
+
+const fontFamilies = {
+  latinAndCyrillic: helmet,
```

Comment:
latinSans and latinSerif arn't scripts, [the list is here](https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/scripts.js)

Every script needs the ability of requesting a sans and serif font family, But also, I dont think it should be based on script, but service. 2 services could use the same script, but for data reasons we dont want to include a custom font.

Reply:
This PR just covers the definition of the font-families and fallbacks. Loading of custom fonts I think would be done in simorgh.